modules/heat_conduction/test/tests/gap_heat_transfer_mortar/closed_gap_prescribed_pressure.i

```diff
@@ -0,0 +1,191 @@
+## Units in the input file: m-Pa-s-K
+
+[Mesh]
+  [left_rectangle]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 40
+    ny = 10
+    xmax = 1
+    ymin = 0
+    ymax = 0.5
+    boundary_name_prefix = moving_block
+  []
+  [left_block]
+    type = SubdomainIDGenerator
+    input = left_rectangle
+    subdomain_id = 1
+  []
+  [right_rectangle]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 40
+    ny = 10
+    xmin = 1
+    xmax = 2
+    ymin = 0
+    ymax = 0.5
+    boundary_name_prefix = fixed_block
+    boundary_id_offset = 4
+  []
+  [right_block]
+    type = SubdomainIDGenerator
+    input = right_rectangle
+    subdomain_id = 2
+  []
+  [two_blocks]
+    type = MeshCollectionGenerator
+    inputs = 'left_block right_block'
+  []
+  [block_rename]
+    type = RenameBlockGenerator
+    input = two_blocks
+    old_block = '1 2'
+    new_block = 'left_block right_block'
+  []
+  [interface_secondary_subdomain]
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = 'fixed_block_left'
+    new_block_id = 3
+    new_block_name = 'interface_secondary_subdomain'
+    input = block_rename
+  []
+  [interface_primary_subdomain]
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = 'moving_block_right'
+    new_block_id = 4
+    new_block_name = 'interface_primary_subdomain'
+    input = interface_secondary_subdomain
+  []
+[]
+
+[Variables]
+  [temperature]
+    initial_condition = 525.0
+  []
+  [temperature_interface_lm]
+    block = 'interface_secondary_subdomain'
+  []
+[]
+
+[AuxVariables]
+  [interface_normal_lm]
+    order = FIRST
+    family = LAGRANGE
+    block = 'interface_secondary_subdomain'
+    initial_condition = 100.0
```

Comment:
Huh, so you are testing a case where the contact pressure is constant throughout. So that's why you wouldn't see any difference in the results with the `_normal_pressure[0]` evaluation.